### PR TITLE
#270 | update git setup docs

### PR DESCRIPTION
**Pull request by hoavt**
connect #270

### CODE REVIEW

**Comment by hoatle** [comment]
check my comment here: https://github.com/teracyhq/dev/issues/270#issuecomment-314052081

**Comment by hoavt** [comment]
@hoatle the doc is updated. Please check.

File: docs/getting_started.rst [hunk 1]
```diff
@@ -377,6 +377,10 @@ Git Setup
 ---------
 
 To work with ``git``, complete the following guides to set up ssh keys:
-https://help.github.com/categories/ssh/
+https://help.github.com/articles/connecting-to-github-with-ssh/
 
-*Congratulations, you’ve all set now!*
+   .. note::
+
+      On Window, you should use ``Cygwin Terminal`` instead of ``Git Bash``.
```
Comment (hoatle) [inline docs/getting_started.rst]: you must always use ``Cygwin Terminal``, not ``Git Bash``.

**Comment by hoavt** [comment]
Applied feedback.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -377,6 +377,10 @@ Git Setup
 ---------
 
 To work with ``git``, complete the following guides to set up ssh keys:
-https://help.github.com/categories/ssh/
+https://help.github.com/articles/connecting-to-github-with-ssh/
 
-*Congratulations, you’ve all set now!*
+   .. note::
+
+      On Window, you must always use ``Cygwin Terminal``, not ``Git Bash``.
```
Comment (hoatle) [inline docs/getting_started.rst]: On Windows

**Comment by hoatle** [comment]
@hoavt review done, please see my comments.

**Comment by hoavt** [comment]
Applied your comments @hoatle

**Comment by hoatle** [comment]
@hoavt what is this file? docs/_static/workflow/error.png

**Comment by hoatle** [comment]
too many commits for this simple change, please squash all commits into one, it's better.

**Pull request closed without merging**